src/components/App/App.js

```diff
@@ -25,7 +26,12 @@ function App() {
             />
           )}
         />
-
+        <Route 
+          exact path='/event-form'
```

Comment:
Just noting that we need to adjust our 'to' prop on the Form's navlink as it currently points to /form and not /event-form